.github/workflows/ci.yml

```diff
@@ -103,9 +103,67 @@ jobs:
       - name: Install node modules
         run: yarn install --frozen-lockfile
       - name: Run integration tests
-        run: yarn integration-tests\
+        run: yarn integration-tests
       # TODO: Set up slack notifications
       # - name: Running size integration tests (failures are reported in Slack only).
       #   run: |
       # If the size integration tests fail, report the failure to a dedicated #components-ci-size-tracking Slack channel.
       # yarn integration-tests:size-test || yarn ci-notify-slack-failure components-ci-size-tracking
+
+  linker_aot_tests:
```

Comment:
Do these linker tests run with free runners? seems like a potential source of slowness